mautic/mautic
# Salesforce activity link

[//]: # ( Please answer the following questions: )

| Q  | A
| --- | ---
| Bug fix? | enhancement
| New feature? | 
| Related user documentation PR URL | https://github.com/mautic/documentation/pull/133
| Related developer documentation PR URL | 
| Issues addressed (#s or URLs) | 
| BC breaks? | 
| Deprecations? | 

[//]: # ( Note that all new features should have a related user and/or developer documentation PR in their respective repositories. )

[//]: # ( Required: )
#### Description:
This PR uses the new timeline view for the push activity object in Salesforce

#### Steps to test this PR:
1. Apply this PR
2. use PR #2871 this fixes some bugs for the contact's activity url
3. use the push activity command to push contact's activities to Salesforce (Eg: `php app/console mautic:integration:pushleadactivity  --integration=Salesforce --time-interval="29 days"`)
URL should change the to the new plugin link for contact's activity.
